File: app/src/main/java/cn/devcxl/photosync/ptp/interfaces/FileTransferListener.java
package cn.devcxl.photosync.ptp.interfaces;

import cn.devcxl.photosync.ptp.usbcamera.BaselineInitiator;
import cn.devcxl.photosync.ptp.usbcamera.Container;

/**
 * Created by rainx on 2017/5/7.
 */

public interface FileTransferListener {
    /**
     *
     * @param bi
     * @param fileHandle
     * @param totalByteLength 文件总长度
     * @param transterByteLength 文件已传输长度
     */
    void onFileTranster(BaselineInitiator bi, int fileHandle, int totalByteLength, int transterByteLength);
}

File: app/src/main/java/cn/devcxl/photosync/ptp/interfaces/FileTransferListener.kt
package cn.devcxl.photosync.ptp.interfaces

import cn.devcxl.photosync.ptp.usbcamera.BaselineInitiator

/**
 * @author devcxl
 */
fun interface FileTransferListener {
    /**
     * @param totalByteLength 文件总长度
     * @param transterByteLength 文件已传输长度
     */
    fun onFileTranster(
        bi: BaselineInitiator,
        fileHandle: Int,
        totalByteLength: Int,
        transterByteLength: Int
    )
}
